devel/poco/Portfile

```diff
@@ -21,41 +22,39 @@ homepage        http://pocoproject.org/
 master_sites    ${homepage}releases/poco-[strsed ${version} {s/[^0-9.].*$//}]/
 distname        ${name}-${version}-all
 
-checksums       rmd160  ff6069681246bafc546a842d92ec627b683b62fb \
-                sha256  17506e0f7abc2a6e933ef0f46a6c5ca6f7cb599370b11979f4e61b604ada240f
+checksums       rmd160  51b6d56429e118014e568443240b0e90a84c517d \
+                sha256  61f1e26e25af2201295b6a58a8e2bf74063ad3bf49c8e969ba08af42310716c2
 
 set docdir      ${prefix}/share/doc/${name}
 
 if {${subport} eq ${name}} {
 use_bzip2       yes
 
-set omit        {NetSSL_OpenSSL Data/ODBC Data/MySQL Crypto}
+set omit        {NetSSL_OpenSSL Data/SQLite Data/ODBC Data/MySQL Crypto}
+
+if {"x86_64" eq ${configure.build_arch}} {
+    set darwin      Darwin64
+} else {
+    set darwin      Darwin
+}
 
 if {[string match *clang* ${configure.compiler}]} {
-    set config      Darwin-clang
+    set config      ${darwin}-clang-libc++
 } else {
-    set config      Darwin
+    set config      ${darwin}
 }
 
 universal_variant   no
-
-# Something in the dependency checking matches the macports.org in the path
-# as a *.o file. It's all downhill from there. This patch removes the
-# dependency (.dep) file creation & use from the Makefiles.
-#
-# Note future maintainers / contributors: you will find you won't need
-# this patch when building outside the *macports.org* path. Don't remove it
-# without testing within the *macports.org* build path. See ticket #19486.
-patchfiles          patch-rules-deps.diff
-
-configure.args      --no-samples --config=${config}
+configure.args      --static --shared --no-samples --unbundled --config=${config}
```

Comment:
Why would you need **--static** and **--shared** on the same line>

Reply:
well, we'd like static AND shared libraries to be built.